src/apolo_app_types/helm/apps/launchpad.py

```diff
@@ -28,29 +28,15 @@
 )
 
 
+PASSWORD_CHAR_POOL = string.ascii_letters + string.digits
+
+
 def _generate_password(length: int = 12) -> str:
     if length < 4:
         err_msg = "Password length must be at least 4"
         raise ValueError(err_msg)
 
-    # At least one from each category
-    lower = random.choice(string.ascii_lowercase)
-    upper = random.choice(string.ascii_uppercase)
-    digit = random.choice(string.digits)
-    special = random.choice("!@#$%^&*()-_=+[]{};:,.<>?/")
```

Comment:
we can make `special` as a condition for `_generate_password`, wdyt?

Reply:
@YevheniiSemendiak it's not a reusable function, and here we won't need special, because keycloak is not escaping password properly